packages/react/src/components/dropdown/select/Select.tsx

```diff
@@ -65,6 +65,7 @@ export interface SelectCustomTheme {
 }
 
 export type CommonSelectProps<OptionType> = {
+  'aria-labelledby'?: string;
```

Comment:
Should we have some description text for this property?

Reply:
- [ ] Add description for aria-labelledby property